hoot-core/src/main/cpp/hoot/core/scoring/MatchComparator.cpp

```diff
@@ -473,10 +473,6 @@ void MatchComparator::_findActualMatches(const ConstOsmMapPtr& in, const ConstOs
       {
         u1.insert(uuidStr);
       }
-      else if (u2.size() == 0 && u1.size() > 0)
-      {
-        u2.insert(uuidStr);
-      }
```

Comment:
Completely unnecessary `else if` since the contents of the `else if` and the `else` are exactly the same.